### ISSUE TITLE
Fix `odo project delete` output to clearly indicate the project name

### ISSUE THREAD

**Issue by valaparthvi** [openshift/odo#5120]
/kind bug

## What versions of software are you using?

**Operating System:**
Fedora 34
**Output of `odo version`:**
odo v2.3.1 (0be3021ee
## How did you run odo exactly?
1. Be logged in or logged out of a cluster (does not matter).
3. `odo project delete non-existent-project`

## Actual behavior
```sh
➜ odo project delete unknown
 ✗  The project unknown does not exist. Please check the list of projects using `odo project list`
```

## Expected behavior
```sh
➜ odo project delete unknown
 ✗  The project 'unknown' does not exist. Please check the list of projects using `odo project list`
```

**Comment by feloy** [comment]
I replaced `default` with `unknown` as the `default` namspace exists

**Comment by valaparthvi** [comment]
/close
Fixed by #5149
